annotation-definitions/src/main/java/eu/europeana/annotation/definitions/model/vocabulary/AnnotationTypes.java

```diff
@@ -8,5 +8,5 @@
 @Deprecated
 public enum AnnotationTypes {
 
-	OBJECT_COMMENT, OBJECT_TAG, IMAGE_ANNOTATION, IMAGE_TAG, SEMANTIC_IMAGE_ANNOTATION, OBJECT_LINKING, OBJECT_TRANSCRIPTION, OBJECT_SUBTITLLE, OBJECT_CAPTION, OBJECT_DESCRIBING
+	OBJECT_COMMENT, OBJECT_TAG, IMAGE_ANNOTATION, IMAGE_TAG, SEMANTIC_IMAGE_ANNOTATION, OBJECT_LINKING, OBJECT_TRANSCRIPTION, OBJECT_SUBTITLLE, OBJECT_CAPTION, OBJECT_DESCRIBING, OBJECT_LINK_FOR_CONTRIBUTING

```

Comment:
the new type of annotation must be added to the templates linked in the description of the create method

Reply:
Please provide me with the link to the document where the new annotation needs to be added, since it seems I cannot find it.

Reply:
it is the [jsonld.jsp](https://github.com/europeana/annotation/blob/develop/annotation-web/src/main/webapp/jsp/template/jsonld.jsp) file which is linked in the description of the create method

Reply:
Ok, updated.